src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -2529,14 +2544,26 @@ Uninstall the following packages: {0}?</value>
     <comment>Preferences | Features | Python | Requires relaunch of Dynamo</comment>
   </data>
   <data name="MessagePackageDepsInBuiltinPackages" xml:space="preserve">
-    <value>{1} has dependencies that conflict with the following Built-In packages : {2}. {0} cannot override packages in the Built-In Packages location.</value>
-  </data>
-  <data name="MessageSamePackageInBuiltinPackages" xml:space="preserve">
-    <value>The package {1} is already installed as a Built-In Package. {0} cannot override packages in the Built-In Packages location.</value>
+    <value>{0} has dependencies that conflict with the following built-in package(s): {1}. 
+    
+Do you wish to continue trying to install {0}?</value>
   </data>
   <data name="MessageSamePackageDiffVersInBuiltinPackages" xml:space="preserve">
-    <value>A different version of the package {1} is already installed as a Built-In Package. {0} cannot override packages in the Built-In Packages location.</value>
+    <value>{0} cannot be installed as it conflicts with a different version of the built-in package, {1}, which is already installed.
+    
+You can try disabling loading packages from built-in package paths, restart {2} and download {0} again.</value>
```

Comment:
I could be remembering incorrectly - but I thought one could also manually mark for unload a built-in package - if thats true, can we also add that to this message as as potential resolution?